common/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/common/model/Address.kt

```diff
@@ -8,20 +8,20 @@ import java.time.LocalDate
 
 data class Address(
   @Schema(description = "The full address on a single line.  No fixed address records will have the fullAddress set to 'No fixed address'.", example = "1 Main Street, Crookes, Sheffield, South Yorkshire, S10 1BP, England")
-  val fullAddress: String,
+  val fullAddress: String? = null,
```

Comment:
We have to make sure every single attribute on the Prisoner record is nullable in order to return a subset of the attributes.